work/Abigail/excercise_1.1.py

```diff
@@ -0,0 +1,14 @@
+import time
+import os
+from os import fork
+
+pid = fork()
+
+if pid < 0:
+    print("Error in fork")
+elif pid == 0: #child process
+    time.sleep(2)
+    print("Hello")
```

Comment:
Its good but is better do exec: 
tip: command = ["python", "-c", "import time; time.sleep(2); print('Hello')"]